class_exercises/main.js

```diff
@@ -0,0 +1,23 @@
+alert('is this working')
+
+//for(var i=0; i<=10; i++){
+//	console.log(i);}
+
+for(var i = 2; i <= 12; i += 2){
+	console.log(i);
+}
+for(var i = 2; i <= 12; i ++){
+	if(i % 2 == 0){
+	console.log(i);
+	}
+}
+
+var friends = [["mayra", "blue"], ["yoli", "yellow"], ["kate", "black"]];
+
+//console.log(`${friends[0][0]} likes color $(friends[0][1]}`);
+
+for(var i=0; i<=2; i++){
```

Comment:
if you use less than friends.length instead of less than or equal to 2 in your for loop conditional, you wont have to update the loop every time you add a new friend to the array!